core/BUILD

```diff
@@ -33,8 +33,8 @@ cc_library(
         "//core/util/logging",
         "@tensorrt//:nvinfer",
     ] + select({
-        ":use_pre_cxx11_abi": ["@libtorch_pre_cxx11_abi//:libtorch"],
-        "//conditions:default": ["@libtorch//:libtorch"],
+        ":use_pre_cxx11_abi": ["@libtorch_pre_cxx11_abi//:libtorch", "@libtorch_pre_cxx11_abi//:c10_cuda"],
+        "//conditions:default": ["@libtorch//:libtorch", "@libtorch//:c10_cuda"],
```

Comment:
Did you forget to mention in the diagnosis that we also need to link with c10_cuda?  :-)

Reply:
We probably don't need that. But it doesn't hurt.